src/languages/sparql.js

```diff
@@ -7,8 +7,8 @@ Description: SPARQL Protocol and RDF Query Language for the semantic web
 Website: https://www.w3.org/TR/sparql11-query/, http://www.w3.org/TR/sparql11-update/, https://www.w3.org/TR/sparql11-federated-query/, http://rawgit2.com/VladimirAlexiev/grammar-diagrams/master/sparql-grammar.xhtml
 */
 
-function(hljs) {
-  var ttl = hljs.getLanguage('ttl').exports;
+module.exports = function(hljs) {
+  var ttl = require("./turtle")(hljs).exports;
```

Comment:
This is a bit strange... traditionally you'd use `getLanguage`, letting us resolve the language at runtime... but that does require someone load the language dependencies correctly (our build tool does this when building a web monolith).  Doing it this way is going to increase the size of the builds since now sparql will include it's own copy of `ttl` instead of just referring to the "already loaded" one.

Was this an intentional change and you're aware of the pros/cons and think we need to build it this way on purpose?

Reply:
Got you. I'm thinking although SPARQL and turtle are closely related that you wouldn't necessarily include them both at all times.

I hadn't considered the size cost of the builds - mostly I was thinking it might feel more natural to have SPARQL be able to be stand-alone, though if this could present a problem I could include some README instruction that for SPARQL to work the user must also register turtle.

I'm open to opinions!

Reply:
> you wouldn't necessarily include them both at all times.

That's reasonable enough logic for me... we can see what the maintainers think.